### ISSUE TITLE
going from list view to dashboards wont update the dashboards with the latest changes.

### ISSUE THREAD

**Issue by MkSafavi** [traggo/server#150]
**Describe the bug**
If you make some changes in the list view (creating and deleting a timespan or stopping a running one), you won't see it in dashboard charts right away. it requires refreshing the whole tab to fetch the new data.
It's a minor inconvenience(issue) that could be by design but I thought I should mention it.  
It's already working properly when you switch from list to calendar.

**To Reproduce**
1. Go to list view
2. Add new time span with manual mode (or stop a running tracker) ![image](https://github.com/traggo/server/assets/50653293/b4764d8a-ed4d-4dbf-80e6-a24f313cd27d)
3. Go to the dashboard
4. Corresponding chart is empty. ![image](https://github.com/traggo/server/assets/50653293/5da4cc91-99d4-4007-b04b-f5dccb6f0dd8)
5. reload the tab
6. chart includes the new entry now ![image](https://github.com/traggo/server/assets/50653293/aeb671af-97b6-4238-87bc-38b7b8a49443)

**Comment by jmattheis** [comment]
Yeah, we probably have to make the dashboards dependent on the timespan mutations, so the cache is flushed.
